enhancements/etcd/disaster-recovery-with-ceo.md

```diff
@@ -19,86 +19,159 @@ see-also:
 
 ## Release Signoff Checklist
 
-- [ ] Enhancement is `implementable`
-- [ ] Design details are appropriately documented from clear requirements
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
 - [ ] Test plan is defined
 - [ ] Graduation criteria for dev preview, tech preview, GA
-- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+- [x] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
 
 ## Summary
 
-Current disaster recovery scripts were created before the introduction of `cluster-etcd-operator` (CEO). Cluster-etcd-operator (CEO) is an operator that handles the scaling of etcd during cluster bootstrap and regular operation and provisioning etcd dependencies such as TLS certificates. This enhancement allows modifications to the DR scripts such that disaster recovery makes use of the presence of CEO.
-
+The Disaster Recovery (DR) scripts were created prior to OCP 4.4 before the introduction of `cluster-etcd-operator` (CEO). Cluster-etcd-operator (CEO) is an operator that handles the scaling of etcd and provisioning etcd dependencies such as TLS certificates. The introduction of CEO obviated the need for some of the disaster recovery scripts such as scaling and TLS certificate generation.
```

Comment:
The summary of this enhancement is to describe how a cluster-admin responds to exceptional circumstances in his cluster, like X, Y, and Z.